internal/disk/lvm.go

```diff
@@ -3,8 +3,14 @@ package disk
 import (
 	"fmt"
 	"strings"
+
+	"github.com/osbuild/osbuild-composer/internal/common"
 )
 
+// Default physical extent size in bytes: logical volumes
+// created inside the VG will be aligned to this.
+const LVMDefaultExtentSize = 4 * common.MebiByte
```

Comment:
Although this is the default, it could change and the manifest could end up having values rounded to a different extent size. It would probably make sense to set the value explicitly when calling lvm tooling in the osbuild stage (ask you mentioned as well when I raised the bug in out team channel). Do you plan a followup?

Reply:
Yeah, indeed. Was the plan for a follow up because for this we also need osbuild changes. 👍